cabal.project

```diff
@@ -6,21 +6,25 @@ packages:
   cardano-crypto-class
   slotting
 
+-- Disable tests in a dependency (which are currently broken)
+package cardano-crypto
+  tests: False
+
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-prelude
-  tag: cb54482c7e3d45f6c1aac490dc937ed835de0333
-  --sha256: 1fg8rkzfxm05bm1mrax5icq1rq9yj7cl70r2114rkpcj2v874q3f
+  tag: fa7a16291d3e93b068d9b13620fc81c88b5600f0
+  --sha256: 1gaj8k5kjdkmnc0l569asc0fwxi0413v1r0piccg8kyscwvhvcwz
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-prelude
-  tag: cb54482c7e3d45f6c1aac490dc937ed835de0333
-  --sha256: 1fg8rkzfxm05bm1mrax5icq1rq9yj7cl70r2114rkpcj2v874q3f
+  tag: fa7a16291d3e93b068d9b13620fc81c88b5600f0
+  --sha256: 1gaj8k5kjdkmnc0l569asc0fwxi0413v1r0piccg8kyscwvhvcwz
   subdir: test
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-crypto
-  tag: 4590efa638397e952a51a8994b5543e4ea3c1ecd
-  --sha256: 0hl2n3bba5v2j0lmxhs7hs01z3aznh2bwf9cb434icq2g0bl8ms3
+  tag: 05aa1bc640c42bfca787531d12595489c1fa3b82
```

Comment:
I would suggest we do not update `cardano-crypto` yet. I suggest we only do that when the ledger folks have the alternate mock VRF ready.